### PR TITLE
Update Chromium data for webRequest Web Extensions interface

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `webRequest` Web Extensions interface. This sets the feature(s) to a version range based upon the date that the feature was added to BCD with the intent of replacing `true` values with ranged values to eliminate `true` values from BCD.

Commit/PR Adding the Feature: #440

### CODE REVIEW

File: webextensions/api/webRequest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "≤44"
```
Comment (rebloor) [inline webextensions/api/webRequest.json:9]: @queengooborg The earliest I can trace this back to is 58 (e.g. https://github.com/mdn/browser-compat-data/pull/324[)](https://github.com/mdn/browser-compat-data/commit/82b72201e181b8aad847f7904f1c08bda567cb64))

File: webextensions/api/webRequest.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/BlockingResponse",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "≤61"
```
Comment (rebloor) [inline webextensions/api/webRequest.json:32]: Should this be 58 re https://github.com/mdn/browser-compat-data/pull/324[)](https://github.com/mdn/browser-compat-data/commit/82b72201e181b8aad847f7904f1c08bda567cb64)

File: webextensions/api/webRequest.json [hunk 3]
```diff
@@ -98,7 +98,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/HttpHeaders",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "≤61"
```
Comment (rebloor) [inline webextensions/api/webRequest.json:101]: Also 58? https://github.com/mdn/browser-compat-data/pull/324[)](https://github.com/mdn/browser-compat-data/commit/82b72201e181b8aad847f7904f1c08bda567cb64)

**Review by rebloor — COMMENTED** [review]
@queengooborg haven't completed a review of all items here, in case I'm missreading the blame but some items seem to be pre-61

File: webextensions/api/webRequest.json [hunk 4]
```diff
@@ -17,7 +17,9 @@
             "firefox_android": {
               "version_added": "48"
             },
-            "opera": "mirror",
+            "opera": {
```
Comment (rebloor) [inline webextensions/api/webRequest.json:20]: @queengooborg what's the reference for this update?

**Comment by rebloor** [comment]
@queengooborg, before I review again, are you happy with all the other updates? The comments I made were only examples of features where the introduction appeared to be earlier.

File: webextensions/api/webRequest.json (hunk 4 above)
Comment (queengooborg) [reply webextensions/api/webRequest.json]: At the time, I hadn't realize that the consistency linter was slightly broken when dealing with ranges, so I made some changes to Opera's data when it should've been Chrome that got updated.  If Opera is set to not mirror any longer (and I don't explicitly mention it), it's _probably_ wrong...!

File: webextensions/api/webRequest.json (hunk 1 above)
Comment (queengooborg) [reply webextensions/api/webRequest.json:9]: Ah, this was set to 44 because of a child feature that's set to 44!

**Comment by queengooborg** [comment]
I glanced through the rest of the changes, and I think that all is good -- I'm happy with the other changes!

**Review by rebloor — APPROVED** [review]

**Pull request merged**